src/myinfo-types.ts

```diff
@@ -0,0 +1,110 @@
+export enum MyInfoSource {
+    GovtVerified = '1',
+    UserProvided = '2',
+    NotApplicable = '3',
+    SingPassVerified = '4',
+}
+
+export enum MyInfoSex {
+    Male = 'M',
+    Female = 'F',
+    Unknown = 'U',
+}
+
+interface IMyInfoMetadata {
+    lastUpdated: string
+    source: MyInfoSource
+    classification: 'C'
+}
+
+interface IMyInfoBasicField extends IMyInfoMetadata {
+    value: string
```

Comment:
one of the MyInfo fields has a nasty bug where the value key is missing. it was reported 1-2 years back but we've not gotten a confirmation that it's been fixed. let me check which field it is and get back to you tomorrow.

```typescript
    value?: string
```